app/Actions/Importer/GatherSections.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Actions\Importer;
+
+use DOMDocument;
+use DOMNodeList;
+use DOMXPath;
+use PhpRfcs\Wiki;
+use RuntimeException;
+use Tidy;
+
+class GatherSections
```

Comment:
Perhaps some comments are in place in this file, so that future contributors know how it works.